src/TimeGutter.jsx

```diff
@@ -1,39 +1,53 @@
 import React from 'react';
-import cn from 'classnames';
 import dates from './utils/dates';
-import localizer from './localizer'
+
+import TimeSliceGroup from './TimeSliceGroup.jsx'
 
 let TimeGutter = React.createClass({
 
   propTypes: {
     step: React.PropTypes.number.isRequired,
+    slices: React.PropTypes.number,
+    now: React.PropTypes.instanceOf(Date).isRequired,
     min: React.PropTypes.instanceOf(Date).isRequired,
-    max: React.PropTypes.instanceOf(Date).isRequired
+    max: React.PropTypes.instanceOf(Date).isRequired,
+    showlabels: React.PropTypes.bool,
+    timesliceClassnames: React.PropTypes.string,
+    culture: React.PropTypes.string,
+    timeGutterFormat: React.PropTypes.string
+  },
+
+  getDefaultProps() {
+    return {
+      step: 10,
+      slices: 2,
+      now: new Date(),
```

Comment:
this is subtlety different logic then the old code, This value is calculated once, whereas the date in `render()` is refreshed often, which means that "now" will only update on a refresh. Admittedly the other one isn't much better, but lets avoid possible bugs unnecessarily.